charts/consul/templates/_helpers.tpl

```diff
@@ -22,13 +22,17 @@ as well as the global.name setting.
             {{ "{{" }}- end -{{ "}}" }}
 {{- end -}}
 
-{{- define "consul.serverTLSCATemplate" -}}
+{{- define "consul.vaultCATemplate" -}}
```

Comment:
extracted `serverTLSCATemplate` to a more generic name. 

modified `serverTLSCATemplate` to call `vaultCATemplate` because there are lots of existing references that would each require a few line code change.